akka-actor-typed/src/main/scala/akka/actor/typed/Behavior.scala

```diff
@@ -244,7 +244,7 @@ object Behavior {
    * INTERNAL API
    */
   @InternalApi private[akka] val unhandledSignal
-    : PartialFunction[(TypedActorContext[Nothing], Signal), Behavior[Nothing]] = {
+      : PartialFunction[(TypedActorContext[Nothing], Signal), Behavior[Nothing]] = {
```

Comment:
seriously, have you still not installed the intellij plugin? or are you secretly using vim? ;-)

Reply:
I have, think it gets confused by switching branches or something, I still haven't quite figured out how/why.